.github/actions/chart/action.yaml

```diff
@@ -6,47 +6,38 @@ inputs:
   keep_previous:
     required: false
     default: no
-  ibs_release:
-    required: false
-    default: no
 
 runs:
   using: composite
-  steps: 
+  steps:
     - name: Install yq
       uses: mikefarah/yq@v4.28.2
     - name: Set image repositories
       id: set_repo
       shell: bash
       env:
         B_ENV: ${{ inputs.build_env }}
-        IBS_RELEASE: ${{ inputs.ibs_release }}
       run: |
-        if [ "${IBS_RELEASE}" == "yes" ]; then
-          echo "build_repo=registry.suse.com" >> $GITHUB_OUTPUT 
-          echo "build_env=release" >> $GITHUB_OUTPUT
+        if [ "${B_ENV}" == "stable" ]; then
+          echo "build_repo=registry.suse.com" >> $GITHUB_OUTPUT
         else
-          echo "build_repo=registry.opensuse.org/isv/rancher/elemental/${B_ENV}/containers" >> $GITHUB_OUTPUT
-          echo "build_env=${B_ENV}" >> $GITHUB_OUTPUT
+          echo "build_repo=registry.opensuse.org/isv/rancher/elemental/${B_ENV}/charts" >> $GITHUB_OUTPUT
         fi
-    - name: Build chart for release
+    - name: Pull chart from OBS
       env:
         B_REPO: ${{ steps.set_repo.outputs.build_repo }}
-        OPERATOR_REPO: rancher/elemental-operator
-        SEEDIMAGE_REPO: rancher/seedimage-builder
-        CHANNEL_REPO: rancher/elemental-teal-channel
+        OPERATOR_CHART: rancher/elemental-operator-chart
+        CRDS_CHART: rancher/elemental-operator-crds-chart
       shell: bash
       run: |
-        REGISTRY_URL=${B_REPO} \
-        REPO=${OPERATOR_REPO} \
-        REPO_SEEDIMAGE=${SEEDIMAGE_REPO} \
-        REPO_CHANNEL=${CHANNEL_REPO} \
-        make chart
+        mkdir -p build
+        CHART_VERSION=$(git describe --abbrev=0 --tags 2>/dev/null | sed 's/[a-z]*\([0-9]\(\.[0-9]\)\{0,2\}\).*/\1/g')
+        helm pull --debug -d build --version ${CHART_VERSION} oci://${B_REPO}/${CRDS_CHART}
```

Comment:
🤔 why the --debug flag only for the crds chart?
Is it because is the same registry it is not worth to have it also for the operator one?

Reply:
Because it is a leftover of debugging it 😓 removing it